src/main/java/edu/kis/vh/nursery/DefaultCountingOutRhymer.java

```diff
@@ -0,0 +1,45 @@
+package edu.kis.vh.nursery;
+
+public class DefaultCountingOutRhymer {
+
+    private static final int SIZE = 12;
+    private static final int INITIAL_VALUE = -1;
+    private static final int FAILURE = -1;
+    private static final int MAX_SIZE = 12;
+
+    private final int[] numbers = new int[SIZE];
```

Comment:
This should be MAX_SIZE (or e.g., capacity)

Reply:
Fixed.